### PR TITLE
(pre-release) update staging build-config

### PR DESCRIPTION
**Pull request by donaldkibet**
## Description

Update build config version for staging in addition also updated script to automate generation of KDOD frontend assets

### CODE REVIEW

**Review by makombe — COMMENTED** [review]
@donaldkibet can we add the labmanifest ?

**Comment by donaldkibet** [comment]
Okay sure

File: dev-build.sh [hunk 1]
```diff
@@ -1,10 +1,13 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (donaldkibet) [inline dev-build.sh:1]: Explicitly have it as bash command

File: dev-build.sh [hunk 2]
```diff
@@ -30,7 +33,21 @@ cp "${CWD}/assets/favicon.ico" "${CWD}/frontend"
 cp "${CWD}/frontend-config/dev/kenyaemr.config.json" "${CWD}/frontend"
 cp "${CWD}/frontend-config/dev/openmrs.config.json" "${CWD}/frontend"
 
-# Function to handle openmrs-esm apps
+# Copy KDOD config or registration config based on user input and update index.html
+if [ "$is_kdod" = "y" ] || [ "$is_kdod" = "Y" ]; then
```
Comment (donaldkibet) [inline dev-build.sh:37]: Dynamically decide which config we want in the generate files. This is to support KDOD

File: frontend-config/dev/kenyaemr.config.json [hunk 3]
```diff
@@ -243,298 +243,6 @@
             }
         ]
     },
```
Comment (donaldkibet) [inline frontend-config/dev/kenyaemr.config.json:245]: Centralized patient-registration into its own file

File: frontend-config/prod/openmrs.config.json [hunk 4]
```diff
@@ -97,6 +97,13 @@
                             ]
                         }
                     },
+                    "procedure-dashboard-link":{
```
Comment (donaldkibet) [inline frontend-config/prod/openmrs.config.json:100]: Ensure we have procedure dashboard displayed according to roles

File: frontend-config/staging/build-config.json [hunk 5]
```diff
@@ -1,42 +1,44 @@
...
+    "@kenyaemr/esm-billing-app": "5.3.3-pre.1048",
+    "@kenyaemr/esm-patient-clinical-view-app": "5.3.3-pre.1048",
     "@kenyaemr/esm-bed-management-app": "1.0.1-pre.21",
     "@kenyaemr/esm-procedure-app": "3.1.1-pre.152",
-    "@kenyaemr/esm-shr-app": "5.2.1-pre.1015",
-    "@kenyaemr/esm-pharmacy-app": "5.2.1-pre.1015"
+    "@kenyaemr/esm-shr-app": "5.3.3-pre.1048",
+    "@kenyaemr/esm-pharmacy-app": "5.3.3-pre.1048",
+    "@kenyaemr/esm-providers-app": "5.3.3-pre.1048",
+    "@kenyaemr/esm-lab-manifest-app":"5.3.3-pre.1048"
```
Comment (donaldkibet) [inline frontend-config/staging/build-config.json:41]: Included lab manifest

**Review by donaldkibet — COMMENTED** [review]
cc @RAJABIBRAZ @andrineM

**Comment by donaldkibet** [comment]
cc @makombe @ojwanganto @andrineM @RAJABIBRAZ @chesterosoronyas @njorocs merging this in to enable testing of this patch

File: .github/workflows/deployment_8500_spa.yml [hunk 6]
```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Execute Development Build Script
         run: |
           chmod +x ./dev-build.sh
-          ./dev-build.sh
+          echo "n" | ./dev-build.sh
```
Comment (Murithijoshua) [inline .github/workflows/deployment_8500_spa.yml:30]: is echo necessary here?

File: staging-build.sh [hunk 7]
```diff
@@ -1,22 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (Murithijoshua) [inline staging-build.sh:1]: This might cause issue when running file as executable i.e ./staging-build.sh, I prefer #!/usr/bin/bash|sh

**Pull request merged**